includes/utilities.inc

```diff
@@ -238,7 +238,7 @@ function islandora_iiif_manifests_get_image_info($object) {
   $handler = new $handler_class($object);
   $handle = $handler->getFullHandle($object);
 
-  $encoded_handle = urlencode('hdl:' . $handle);
```

Comment:
Little strange change for this PR. Are you really sure the "hdl:" needs to be removed? I think cantaloupe expects a hdl: before the handle, see delegates.rb on cantaloupe server